src/components/atoms/ButtonBuy.tsx

```diff
@@ -23,6 +26,40 @@ interface ButtonBuyProps {
   type?: 'submit'
 }
 
+function getConnectivityHelpText(
```

Comment:
This is returning an actual React component, not just text string so naming and code style should be adapted accordingly where this method here should be a component like `ConnectivityHelp` with props instead of params.